cli_tools/gce_vm_image_import/importer/processor.go

```diff
@@ -37,24 +42,62 @@ type processorProvider interface {
 
 type defaultProcessorProvider struct {
 	ImportArguments
-	imageClient   createImageClient
+	computeClient daisyCompute.Client
 	diskInspector disk.Inspector
 }
 
 func (d defaultProcessorProvider) provide(pd persistentDisk) (processor, error) {
 	if d.DataDisk {
-		return newDataDiskProcessor(pd, d.imageClient, d.Project,
+		return newDataDiskProcessor(pd, d.computeClient, d.Project,
 			d.Labels, d.StorageLocation, d.Description,
 			d.Family, d.ImageName), nil
 	}
+	var err error
+	pd, err = d.inspectDisk(pd)
```

Comment:
inspection doesn't naturally belog to provide() method. provide() is a processor factory method. Inspection should either be a separate step or a part of the processor.

Reply:
Sounds great, I moved it into bootableDiskProcessor.process()